resources/options_lookup.tsv

```diff
@@ -998,11 +998,11 @@ Insulation Unfinished Attic	"Roof R-49, Unvented"	ResidentialConstructionsUnfini
 Insulation Finished Roof	None																																														
 Insulation Finished Roof	Uninsulated	ResidentialConstructionsFinishedRoof	cavity_r=0	install_grade=1	cavity_depth=9.25	filled_cavity=false	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
 Insulation Finished Roof	R-7	ResidentialConstructionsFinishedRoof	cavity_r=7	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
-Insulation Finished Roof	R-13	ResidentialConstructionsFinishedRoof	cavity_r=13	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
+Insulation Finished Roof	R-13	ResidentialConstructionsFinishedRoof	cavity_r=13	install_grade=1	cavity_depth=4.01	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					

```

Comment:
@JLReyna Typo? I think this should be cavity_depth=3.5

Reply:
Good catch. That was the calculated version. BEopt uses 9.25

Reply:
I see 3.5 in BEopt for R-13